android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/components/EmojiRatingAnswer.kt

```diff
@@ -0,0 +1,103 @@
+package co.nimblehq.ic.kmm.suv.android.ui.components
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import co.nimblehq.ic.kmm.suv.android.ui.theme.Typography
+import co.nimblehq.ic.kmm.suv.android.util.EmojiParameterProvider
+
+const val THUMPS_UP_EMOJI = "\uD83D\uDC4D\uD83C\uDFFB" // 👍🏻
+const val HEART_EMOJI = "❤️"
+const val STAR_EMOJI = "⭐️️"
+const val POUTING_FACE_EMOJI = "\uD83D\uDE21" // 😡
+const val CONFUSED_FACE_EMOJI = "\uD83D\uDE15" // 😕
+const val NEUTRAL_FACE_EMOJI = "\uD83D\uDE10" // 😐
+const val SLIGHTLY_SMILING_FACE_EMOJI = "\uD83D\uDE42" // 🙂
+const val GRINNING_FACE_WITH_SMILING_EYES_EMOJI = "\uD83D\uDE04" // 😄
```

Comment:
I'd suggest naming those constants starting with EMOJI_XXXXX_XXXXX, it would be helpful to group the category of constants.

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/116/commits/949ea88c53e236ecb61123ff53995043f6d52189